### PR TITLE
Case without limiters

### PR DESCRIPTION
**Pull request by crocicc**
## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by trontrytel** [comment]
This PR will add a SB2006 parameters struct without limiters. We will use it for dispatch. Things left to do:

- add a test based on https://github.com/CliMA/CloudMicrophysics.jl/blob/0985330de6093ac5d4efe62dfff607c937fa6478/test/test_SB2006_limiters.jl
- double check the typos in effective radius
- think some more about the epsilons in Z and r_eff

File: src/Microphysics2M.jl [hunk 1]
```diff
@@ -46,22 +46,85 @@ end
...
- - `N_rai` raindrops number density
+ - `N_rai` rain drop number concentration
 
-    Returns the mean mass of raindrops, xr, the rate parameter of the assumed size distribution
-    of raindrops (based on drops diameter), λr, limited within prescribed ranges, and the two 
-    rain distribution parameters, A and B.
+    Returns the mean mass of rain drops xr,
+    the rate parameter of the assumed size distribution of raindrops
+    (as a function of drop diameter) λr, limited within prescribed ranges,
+    and the two rain distribution parameters, A and B.
```
Comment (sajjadazimi) [inline src/Microphysics2M.jl:95]: It would be helpful to write the distribution in the doc to see what A and B represent.

**Comment by sajjadazimi** [comment]
Thank you so much. This looks good to me. I had a minor suggestion about the doc. Also, I noticed when q and rho are zero radar reflectivity (from CM1) returns Inf. Please check that too.

**Comment by crocicc** [comment]
> Thank you so much. This looks good to me. I had a minor suggestion about the doc. Also, I noticed when q and rho are zero radar reflectivity (from CM1) returns Inf. Please check that too.

Thank for reporting this. We'll fix it!

**Review by trontrytel — APPROVED** [review]

**Review by sajjadazimi — APPROVED** [review]

**Comment by sajjadazimi** [comment]
Thank you for the work! It look good to me.

**Pull request merged**